docs/developer-guide/rules.md

```diff
@@ -1,57 +1,81 @@
-# Writing rules
+# Working on rules
+
+**Please help us create, enhance, and debug stylelint rules!**
+
+There are well over a hundred rules already, so stylelint *needs* community contributions to continue its growth.
+
+If you like stylelint and open source software (since you're reading this, you almost certainly do), please consider taking some time to pitch in. Not only will you help stylelint thrive, you will also learn a thing or two — about CSS, PostCSS, Node, ES2015, unit testing, open source software, and more.
+
+**We want to do everything we can to encourage contributions!** So if you want to participate but don't end up doing it for one reason or another, please file an issue and give us feedback about what we could do to better encourage you.
+
+Also: we hope that your participation in the project isn't a one-off. *We'd love to add more members to the organization and see more regulars pop up in issues and pull requests!*
+
+## Creating new rules
+
+### Let everyone know
+
+First, open [an issue](https://github.com/stylelint/stylelint/issues/new) and let everyone else know that you intend to create a new rule.
+
+Usually we have some discussion about the rule's purpose, name, and options before it's ready for serious development.
+
+Once you have something to show, you'll create a [pull request](https://github.com/stylelint/stylelint/compare) to continue the conversation.
+
+### Come up with a name
 
 Have a look at the [rules guide](/docs/user-guide/rules.md) to familiarize yourself the rule naming conventions.
```

Comment:
Shall we expand on this?

"Have a look at the [rules guide](/docs/user-guide/rules.md) to familiarize yourself the rule naming conventions. In particular, the section [about rule names](https://github.com/stylelint/stylelint/blob/contribution-docs/docs/user-guide/rules.md#about-rule-names). We take care to ensure that all the rules are named consistently, so that ..." (you said something about this a couple of weeks back, but I can't find it. Something about a good user experience)...